portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -32,26 +38,44 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/** Servlet responsible for creating tasks. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  
+  /** Takes in comment data and creates tasks that contain those comments in the
+  language specified by the user */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Task").addSort("timestamp", SortDirection.DESCENDING);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Translate translate = TranslateOptions.getDefaultInstance().getService();
+
     PreparedQuery results = datastore.prepare(query);
 
+    // Grabs the desired comment language from the user
+    String language = getParameter(request, "languageCode", "en");
+
+    // Adds comment data to the list of tasks
     List<Task> tasks = new ArrayList<>();
     for (Entity entity : results.asIterable()) {
       long id = entity.getKey().getId();
-      String comment = (String) entity.getProperty("comment");
+      String og_comment = (String) entity.getProperty("comment");
       long timestamp = (long) entity.getProperty("timestamp");
+
+      /* Translates the comments and creates a task element that contains
```

Comment:
Another nitpick - For inline comments explaining implementation most code bases use `//` comments.